test/test.tf

```diff
@@ -57,4 +58,5 @@ module "test_gateway" {
   api_aliases     = "${local.api_aliases}"
   certificate_arn = "${aws_acm_certificate_validation.cert.certificate_arn}"
   lb_fqdn         = "${local.lb_fqdn}"
+  mdw_fqdn        = "${local.mdw_fqdn}"
```

Comment:
You need to have this domain added in the certificate_arn

Reply:
no